tests/test_atari.py

```diff
@@ -62,7 +62,7 @@ def test_deepq():
     env = wrap_atari_dqn(env)
 
     model = DQN(env=env, policy=CnnPolicy, learning_rate=1e-4, buffer_size=10000, exploration_fraction=0.1,
-                exploration_final_eps=0.01, train_freq=4, learning_starts=10000, target_network_update_freq=1000,
+                exploration_final_eps=0.01, train_freq=4, learning_starts=100, target_network_update_freq=100,
```

Comment:
`target_network_update_freq` should probably be lower than `learning_starts` so we have some updates going on. Maybe 100 for `learning_starts` and 50 for `target_network_update_freq`?

Reply:
not necessary of the `num_timesteps` is big enough, no?